comid/rawint.go

```diff
@@ -0,0 +1,344 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package comid
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/veraison/corim/encoding"
+	"github.com/veraison/corim/extensions"
+)
+
+var (
+	ErrRawIntUnsetRawInt = errors.New("RawInt value unset")
+	ErrRawIntEmpryInput  = errors.New("Empry input")
```

Comment:
Exported symbols must be documented (comment directly above starting with the symbol name)